src/ripe/bulk_order.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+class BulkOrderAPI(object):
+
+    def get_bulk_orders(self, **kwargs):
+        url = self.base_url + "bulk_orders"
+        contents = self.get(url, **kwargs)
+        return contents
+
+    def get_bulk_order(self, number, **kwargs):
+        url = self.base_url + "bulk_orders/%d" % number
+        contents = self.get(url, **kwargs)
+        return contents
+
+    def set_bulk_order_status(self, number, status, **kwargs):
+        url = self.base_url + "bulk_orders/%d/%s" % (number, status)
+        contents = self.put(url, **kwargs)
+        return contents
+
+    def create_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "create", **kwargs)
+
+    def produce_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "produce", **kwargs)
+
+    def quality_assure_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "quality_assure", **kwargs)
+
+    def reject_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "reject", **kwargs)
+
+    def ready_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "ready", **kwargs)
+
+    def send_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "send", **kwargs)
+
+    def block_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "block", **kwargs)
+
+    def receive_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "receive", **kwargs)
+
+    def return_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "return", **kwargs)
+
+    def cancel_bulk_order(self, number, **kwargs):
+        return self.set_bulk_order_status(number, "cancel", **kwargs)
+
+    def import_bulk_order(self, name, orders, **kwargs):
+        return self._import_bulk_order(name, orders, **kwargs)
+
+    def attachments_bulk_order(self, number, **kwargs):
+        url = self.base_url + "bulk_orders/%d/attachments" % number
+        contents = self.get(url, **kwargs)
+        return contents
+
+    def create_attachment_bulk_order(self, number, files, **kwargs):
+        url = self.base_url + "bulk_orders/%d/attachments" % number
+        data_m = dict(files = files)
+        contents = self.post(url, data_m = data_m, **kwargs)
+        return contents
+
+    def _import_bulk_order(self, name, orders, **kwargs):

```

Comment:
why the separate method?

Reply:
Following JS SDK's logic: 
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/bulk-order.js#L428
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/bulk-order.js#L532

And to a certain extent the order module too:
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/order.js#L1711
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/order.js#L2001

Reply:
let's not copy these practices from RIPE SDK as it has a very specific way of handling code structure